docs/sources/developers/plugins/package-a-plugin.md

```diff
@@ -44,4 +44,58 @@ There are three steps needed to package a plugin:
 
 The best way to share your plugin with the world is to publish it on [Grafana Plugins](https://grafana.com/plugins). By having your plugin published on Grafana.com, more users will be able to discover your plugin.
 
-To publish a plugin to [Grafana Plugins](https://grafana.com/grafana/plugins), create a pull request to the [Grafana Plugin Repository](https://github.com/grafana/grafana-plugin-repository). Please note that both the source code and the packaged plugin archive need to be publicly available.
+Before you submit your plugin, we ask that you read our guidelines and frequently asked questions.
+
+### Guidelines
+
+To speed up the time it takes to review your plugin:
+
+- Check that your plugin is ready for review using the [plugin validator](https://github.com/grafana/plugin-validator).
+- Read our [6 tips for improving your Grafana plugin before you publish](https://grafana.com/blog/2021/01/21/6-tips-for-improving-your-grafana-plugin-before-you-publish/).
+
+### Frequently Asked Questions
+
+**Do I need to submit a private plugin?**
+
+- No. Please only submit plugins that you wish to make publicly available for the Grafana community.
+
+**How long does it take to review my submission?**
+
+- We're not able to give an estimate at this time, though we're constantly working on improving the time it takes to review a plugin.
+
+## Publishing your plugin for the first time
```

Comment:
**Question:** is H2 `##` intended here?

Probably it is, only asking it to be sure.

Reply:
@leventebalogh good catch, in this place it is intended indeed. But to make the structure more logical I have changed the previous h2 `Publish your plugin on Grafana.com` to h1.